docs/UserGuide.md

```diff
@@ -148,6 +148,20 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Adding a course: `course-add`
+
+Adds a course to TAHub Contacts.
```

Comment:
Not sure if the edit, delete methods etc should also be added into the User Guide?